spec/features/scheduling/schedule_unpublishable_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```

Comment:
I think this one should still be `schedule_requirements_spec`. For me, 'unpublishable' is pretty ambiguous and my first thought was that we would never be able to publish it for some reason :-|.